docker-compose-dev.yml

```diff
@@ -0,0 +1,51 @@
+version: '3'
+# usage:
+# (sudo) docker-compose up -d
+# (sudo) docker-compose down
+services:
+
+  mongodb:
+    image: mvertes/alpine-mongo
+    container_name: mongodb
+    ports:
+      - '27017:27017'
+    expose:
+      - '27017'
+    networks:
+      - scout-net
+
+  scout-cli:
+    build: .
```

Comment:
Was this one (and the same change later) the only difference from the non-dev docker-compose?

Reply:
Yep. So far. It might be possible to do an editable install, with mounting from outside the container, but lets play it safe with small rebuilds for now.

Reply:
As long as requirements are not touched, the cached rebuilds should be fast.